# tGCN
Flexible Protocol for Targeted Gene Co-expression Network Analysis

The "Flexible Protocol for Targeted Gene Co-expression Network Analysis" (Coman et al. 2014) presents a user-friendly workflow to perform targeted gene co-expression network analyses using large-scale gene expression data and explains critical steps from design and data analysis to prediction of functionally related modules.

Highlights:

    #infers statistically significant gene co-expression networks
    uses quantitative data generated by RNA-seq, microarrays, RT-qPCR, proteomics etc. for any organism
    runs on various platforms (Linux, Mac, Windows)
    uses exclusively open source software (R, Cytoscape)
    
Updates:

    an additional CytoscapeInput.R script is now provided 
    prepares the output files from t_GCN.R for import into Cytoscape
    uses any t_GCN.R html output file
    converts the matrix with p-values corresponding to correlation coefficients into Cytoscape format:
    column1 = target gene identifier                                   
    column2 = interaction (i.e. adjusted p-value calculated by t.GCN.R) 
    column3 = query gene identifier   
    optional: selects only significant correlations (i.e. p-value <= 0.05)
    saves the files, which can be readily imported into Cytoscape
